tests/suites/test_suite_psa_crypto_storage_format.function

```diff
@@ -193,24 +122,11 @@ static int can_exercise(const psa_key_attributes_t *attributes)
         return 0;
     }
 #endif
+
     if (is_accelerated_rsa(alg) &&
         (hash_alg == PSA_ALG_RIPEMD160 || hash_alg == PSA_ALG_SHA_384)) {
         return 0;
     }
-#if defined(MBEDTLS_PSA_ACCEL_ALG_RSA_OAEP)
```

Comment:
Note: this change (that part about RSA OAEP) is not reflected in the commit message, but seems correct though.